contracts/lifecycle/DestructibleDelayed.sol

```diff
@@ -0,0 +1,74 @@
+pragma solidity ^ 0.4.18;
+
+import "../ownership/Ownable.sol";
+
+/**
+ * @title DestructibleDelayed
+ * @dev Contract that can be selfdestructed after a delay period.
+ */
+contract DestructibleDelayed is Ownable {
+
+   // Whether a request for the self-destruction of the contract was requested.
+   bool public destructionRequested = false;
+
+   // When self-destruction is possible
+   uint256 public destructionTime = 2**256-1;
```

Comment:
Is this a block number? A timestamp? (in what format?)

Reply:
Timestamp (UNIX)  - seconds. Added a comment